azure-pipelines.yml

```diff
@@ -139,6 +139,9 @@ jobs:
     steps:
       - template: ci/report-start.yml
       - checkout: self
+      - bash:
+          for shmid in $(ipcs -m | sed 1,3d | awk '{print $2}' | sed '$d'); do ipcrm -m $shmid; done
```

Comment:
Can we pull this out into a script which can also be run locally easily when things go to hell? (Perhaps one with a useful name like `release-shared-memory.sh`?)